### PR TITLE
FI-750: Support canonical urls in capabilities test

### PR DESCRIPTION
**Pull request by Jammjammjamm**
This fixes a problem where warnings were being given for unsupported profiles because canonical urls weren't being recognized.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label
- [x] Code diff has been reviewed for extraneous/missing code
- [x] Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

**Comment by okeefm** [comment]
Closes https://github.com/onc-healthit/inferno/issues/451

File: lib/modules/us_core_guidance/us_core_r4_capability_statement_sequence.rb [hunk 1]
```diff
@@ -77,7 +77,13 @@ def json_formats
           'http://hl7.org/fhir/us/core/StructureDefinition/pediatric-weight-for-height',
           'http://hl7.org/fhir/us/core/StructureDefinition/us-core-pulse-oximetry',
           'http://hl7.org/fhir/us/core/StructureDefinition/us-core-smokingstatus',
-          'http://hl7.org/fhir/StructureDefinition/vitalsigns'
```
Comment (okeefm) [inline lib/modules/us_core_guidance/us_core_r4_capability_statement_sequence.rb:80]: What does adding these profiles do?

Comment (Jammjammjamm) [reply lib/modules/us_core_guidance/us_core_r4_capability_statement_sequence.rb:80]: Warnings will be generated for any of these profiles which are not listed as supported in the CapabilityStatement.

Comment (okeefm) [reply lib/modules/us_core_guidance/us_core_r4_capability_statement_sequence.rb:80]: Makes sense.
